package.json

```diff
@@ -37,7 +37,7 @@
     "@lion/radio-group": "^0.21.1",
     "@lion/select": "^0.17.1",
     "@lion/textarea": "^0.16.1",
-    "@pega/constellationjs": "SDK-8.8.2",
+    "@pega/constellationjs": "SDK-23.1.1",
```

Comment:
Use ~23.1.1 version instead of SDK tag

Reply:
Doesn't match SDK versioning rules.  Last digit starts at 10 to avoid confusion with infinity patch version.